### PR TITLE
Codig guidelines - completed for ISO2DFD - Please review and merge

### PR DESCRIPTION
**Pull request by Alberto-Villarreal**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue) 

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Sample Migration (Moving sample from old repository after completing checklist established)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [ X] Implement coding guidelines and ensure code quality.
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [ ] Adhere to readme template 
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ X] Enforce format via clang-format config file
- [X ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ X] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [ ] Tested using Dev Cloud when applicable
- [ ] Implement fixes for ONSAM Jiras
- [ ] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

**Comment by Alberto-Villarreal** [comment]
Please review (coding guidelines) and merge

**Review by JoeOster — DISMISSED** [review]

File: DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim.cpp [hunk 1]
```diff
@@ -111,23 +111,23 @@ void ParticleMotion(queue& q, size_t seed, float* particle_X, float* particle_Y,
...
+    mkl::rng::generate(distr, engine, n_moves, random_X_buf);
+    mkl::rng::generate(distr, engine, n_moves, random_Y_buf);
 
     // Submit command group for execution
     q.submit([&](handler& h) {
       auto a_particle_X = b_particle_X.get_access<access::mode::read_write>(h);
       auto a_particle_Y = b_particle_Y.get_access<access::mode::read_write>(h);
-      auto a_random_X = b_random_X.get_access<access::mode::read>(h);
-      auto a_random_Y = b_random_Y.get_access<access::mode::read>(h);
+      auto random_X_a = random_X_buf.get_access<access::mode::read>(h);
```
Comment (praveenkk123) [inline DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim.cpp:129]: Accessors syntax not changes here. Please change the accessors syntax as per the guidelines

File: DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/src/iso2dfd.cpp [hunk 2]
```diff
@@ -304,14 +304,14 @@ int main(int argc, char* argv[]) {
         //    swaps their content at every iteration.
         if (k % 2 == 0)
           h.parallel_for(global_range, [=](id<2> it) {
```
Comment (praveenkk123) [inline DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/src/iso2dfd.cpp:306]: in parallel_for function use "auto" for id as per the guidelines

Comment (Alberto-Villarreal) [reply DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/src/iso2dfd.cpp]: Code updated. Thanks for the suggestion

File: DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim.cpp (hunk 1 above)
Comment (Alberto-Villarreal) [reply DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim.cpp:129]: The "particle diffusion" sample was updated (including the new coding guidelines) in a separate MR # 203. Thanks

**Review by praveenkk123 — APPROVED** [review]

**Review by JoeOster — APPROVED** [review]

**Pull request merged**